pkg/middleware/auth/auth.go

```diff
@@ -265,6 +265,10 @@ func isValidAPIToken(ctx context.Context, dbClient ent.Client, token string) (*a
 
 // getSubjectName returns the subject name for the user
 func getSubjectName(user *ent.User) string {
+	if user == nil {
+		return ""
```

Comment:
instead of returning empty wouldn't we want to progress down the same function setting subjectName to display name, etc.?

Reply:
if the `user` is `nil` we wouldn't have any information, we do fall back to display name below. This wasn't an actual issue we hit but I thought it was/had nil pointer errors I was trying to fix.